src/hooks/useDiscover.ts

```diff
@@ -120,6 +123,14 @@ const useDiscover = <
     );
   }
 
+  if (settings.currentSettings.hideBlacklisted && hideBlacklisted && (hasPermission(Permission.ADMIN) || hasPermission(Permission.MANAGE_BLACKLIST))) {
```

Comment:
`hasPermission(Permission.MANAGE_BLACKLIST)` is enough, as an admin has all permissions (including MANAGE_BLACKLIST)
```suggestion
  if (settings.currentSettings.hideBlacklisted && hideBlacklisted && hasPermission(Permission.MANAGE_BLACKLIST)) {
```